### PR TITLE
Resolve "Move Panix Build Container to Appropriate Repo"

### LINKED ISSUE #222: Move Panix Build Container to Appropriate Repo
Also means `.scuba.yml` will need to be updated to reference the new location and version.

### PR DESCRIPTION
**Pull request by Kfeavel**
Also update tasks.cpp to fix compiler error due to version update.

Closes #222

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,6 @@ tests/run_tests
 
 # Serial Output
 com1.txt
+
+# Doxygen output
+docs/**
```
Comment (micahswitzer) [inline .gitignore]: Add extra newline

File: kernel/sys/tasks.cpp [hunk 2]
```diff
@@ -13,15 +13,8 @@
...
-// this is a literal hack to specifically circumvent validation checks
-// it seems to work fine for the specific intrinsic we need
-// we should probably investigate this further at some point...
-#define _X86INTRIN_H_INCLUDED
-#define __X86INTRIN_H
-#include <ia32intrin.h> // needed for __rdtsc
-#undef __X86INTRIN_H
-#undef _X86INTRIN_H_INCLUDED
+#include <stdint.h>         // Data type definitions
+#include <x86gprintrin.h>   // needed for __rdtsc
```
Comment (micahswitzer) [inline kernel/sys/tasks.cpp:17]: Why did we need this change?

Comment (Kfeavel) [reply kernel/sys/tasks.cpp:17]: The hack you had in place stopped working when GCC was updated from 10 to 11.

Comment (micahswitzer) [reply kernel/sys/tasks.cpp:17]: Oh lol

File: .gitignore (hunk 1 above)
Comment (Kfeavel) [reply .gitignore]: Fixed and rebased.

**Review by micahswitzer — APPROVED** [review]

**Pull request merged**